queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/name_request.py

```diff
@@ -33,9 +33,12 @@ def consume_nr(business: Business, filing: Filing, filing_type: str = None):
 
             namex_svc_url = current_app.config.get('NAMEX_API')
             token = AccountService.get_bearer_token()
+            if flags and (flag_on := flags.is_on('namex-nro-decommissioned')):
+                logger.debug('namex-nro-decommissioned flag: %s', flag_on)
```

Comment:
Do you want to keep this debug statement?

Reply:
not really, kept it there to make sure we remove this once we go to production

Reply:
Sounds good.